HerPublicWebsite.BusinessLogic/Services/RegularJobs/ReferralFollowUpService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using HerPublicWebsite.BusinessLogic.Models;
+using HerPublicWebsite.BusinessLogic.Services.WorkingDayHelper;
+
+namespace HerPublicWebsite.BusinessLogic.Services.ReferralFollowUp;
+
+public interface IReferralFollowUpService
+{
+    public Task<IList<ReferralRequest>> GetReferralsPassedTenWorkingDayThresholdWithNoFollowUp();
+}
+
+public class ReferralFollowUpService : IReferralFollowUpService
+{
+    private readonly IDataAccessProvider dataProvider;
+    private readonly CsvFileCreator.CsvFileCreator csvFileCreator;
+    private readonly IWorkingDayHelperService workingDayHelperService;
+
+    public ReferralFollowUpService(
+        IDataAccessProvider dataProvider,
+        CsvFileCreator.CsvFileCreator csvFileCreator, 
+        IWorkingDayHelperService workingDayHelperService)
+    {
+        this.dataProvider = dataProvider;
+        this.csvFileCreator = csvFileCreator;
+        this.workingDayHelperService = workingDayHelperService;
+    }
+
+    public async Task<IList<ReferralRequest>> GetReferralsPassedTenWorkingDayThresholdWithNoFollowUp()
+    {
+        var endDate = await workingDayHelperService.AddWorkingDaysToDateTime(DateTime.Today, -10);
```

Comment:
In future it would probably be better to make this configurable, in case the threshold changes, but it's fine to leave this time :)

Reply:
Ah good point! Ok, I'll leave it for this PR.